src/content/docs/adventure/sound.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Sound
+slug: adventure/sound
+description: A guide to playing sound with Adventure.
+---
+
+Adventure contains an API to play any built-in or resource pack-provided sound. Note that
+not all platforms implement playing sound.
+
+## Constructing a Sound
+
+Sounds are composed of:
+  * A Key (also known as `Identifier` or `ResourceLocation`) that decides which sound to play. Any custom sounds from resource packs can be used. If a client does not know about sounds, it will ignore the sound (though a warning will be printed to the client log).
+  * A Sound source, used to tell the client what type of sound its hearing. The clients sound settings are also attributed to a source.
+  * A number, determining the radius where the sound can be heard
+  * A number from 0 to 2 determining the pitch the sound will be played at
+
+**Examples:**
+
+```java
+// Create a built-in sound using standard volume and pitch
+Sound musicDisc = Sound.sound(Key.key("music_disc.13"), Sound.Source.MUSIC, 1f, 1f);
+
+// Create a sound from our resource pack with a higher pitch
+Sound myCustomSound = Sound.sound(Key.key("adventure", "rawr"), Sound.Source.AMBIENT, 1f, 1.1f);
+```
+
+## Playing a Sound
+
+:::caution
+
+The client can play multiple sounds at once, but as of version 1.16 is limited to 8 sounds playing at once.
+
+In 1.15.2-1.16.5, due to `MC-138832`, the volume and pitch of sounds played with an emitter are ignored.
```

Comment:
Can these link to the JIRA? In the old docs we had a placeholder that could do it for us just with the bug id